Signal/src/view controllers/MessagesViewController.m

```diff
@@ -918,6 +925,47 @@ - (OWSDisplayedMessageCollectionViewCell *)loadErrorMessageCellForMessage:(TSMes
 
 #pragma mark - Adjusting cell label heights
 
+
+/**
+ Due to the usage of JSQMessagesViewController, and it non-conformity to Dynamyc Type
+ we're left to our own devices to make this as usable as possible.
+ JSQMessagesVC also does not expose the constraint for the input toolbar height nor does it seem to
+ give us a method to tell it to re-adjust (I think it should observe the preferredDefaultHeight property).
```

Comment:
I'm tempted to merge this as-is, and then once it's upstreamed we can clean this up along with the JSQMVC update.

Sound good?